src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift

```diff
@@ -134,13 +144,20 @@ final class OnboardingInfoViewController: UIViewController {
 		present(alert, animated: true, completion: nil)
 	}
 
-	private func updateUI() {
+	private func updateUI(exposureManagerState: ExposureManagerState) {
 		guard isViewLoaded else { return }
 		guard let onboardingInfo = onboardingInfo else { return }
 
 		titleLabel.text = onboardingInfo.title
 
-		imageView.image = UIImage(named: onboardingInfo.imageName)
+		let exposureNotificationsNotSet = exposureManagerState.status == .unknown
+		let exposureNotificationsEnabled = exposureManagerState.enabled
+		let exposureNotificationsDisabled = !exposureNotificationsEnabled && !exposureNotificationsNotSet
```

Comment:
This logic is wrong... if you want to the state if risk identification is turned active or not looking only on enabled and ENStatus different from unknown is not enough.. Consider the ENStatus Bluetooth off ... then it still shows active tracing but tracing is clearly not active in this case

Reply:
Based on the discussion we have on the phone, this is fine in most cases. There's one edge case that I found during testing while playing with bluetooth toggle is, that when user activates the tracing during onboarding, and then in the next step he goes to settings and turns bluetooth off, then goes back, he should be asked to enable bluetooth (or to decline), and he wasn't. 

Now it's fixed with [this commit](https://github.com/corona-warn-app/cwa-app-ios/pull/517/commits/73e8946b88b50ec123c265a121d90691ec23db21).

Reply:
Yes, for now its fine, still I find the variable naming not the best :D ... but I'll approve it now because the functionality is correct now and we need to support reset functionality in some way ... I guess it might need some refactoring in the future ;)